Http.cs

```diff
@@ -106,6 +82,38 @@ public static HttpRequest Get(string uri)
 			return new HttpRequest(UnityWebRequest.Get(uri));
 		}
 
+		/// <summary>
+		/// Creates a HttpRequest configured for HTTP GET.
+		/// </summary>
+		/// <param name="uri">The URI of the resource to retrieve via HTTP GET.</param>
+		/// <returns>A HttpRequest object configured to retrieve data from uri.</returns>
+		public static HttpRequest GetTexture(string uri)
+		{
+			return new HttpRequest(UnityWebRequestTexture.GetTexture(uri));
+		}
+
+		/// <summary>
+		/// Create a HttpRequest configured to send json data to a server via HTTP POST.
+		/// </summary>
+		/// <param name="uri">The target URI to which json data will be transmitted.</param>
+		/// <param name="json">Json body data.</param>
+		/// <returns>A HttpRequest configured to send json data to uri via POST.</returns>
+		public static HttpRequest PostJson(string uri, string json)
```

Comment:
I'd put this after the standard `Post` method, similar to the convention you've used with `Get` where the modified version follows the standard version.

Reply:
The issue wit that is the signature would matches the standard `Post(string uri, string postData)`, the difference being content-type. Unity defaults to "application/x-www-form-urlencoded".

Reply:
Done